frontend/src/SendForm.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+
+import UseFormSender from './UseFormSender.js';
+
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import CircularProgress from '@mui/material/CircularProgress';
+import Typography from '@mui/material/Typography';
+
+const SendForm = () => {
+    const [inputValue, setInputValue] = useState('');
+    const { error, loading, cb, data } = UseFormSender('POST', 'http://localhost:8080/api/v1/documents');
```

Comment:
The only comment here is that IMO it's better to rename `cb` to e.g. `postDocument` in here
```suggestion
    const { error, loading, cb: postDocument, data } = UseFormSender('POST', 'http://localhost:8080/api/v1/documents');
```